warhorn/default_config.xml

```diff
@@ -3,8 +3,8 @@
 <!-- To read full documentation for this file, please refer to user_generated/config_sample.xml -->
 
 <data>
-    <warhorn name="Warhorn">
-        <dependency name="jira" install="no" user="no"/>
+    <warhorn name="Warhorn" dependency_filepath="Provide filepath, where version details for packages below are defined. By default the versions defined in requirements.txt at the base directory would be used">
+        <dependency name="jira" install="yes" user=""/>
```

Comment:
Hi @brettleehari,

Why was the "jira" dependency set to yes? We do not need the jira package any more - migrated to requests package many versions ago.

Reply:
It was a change while testing the ticket. Shall change